app/src/main/AndroidManifest.xml

```diff
@@ -19,6 +19,9 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".DetailsActivity"
+            android:exported="true">
```

Comment:
why exported?

Reply:
I put it on to see what the screen would look like without running the application flow, I forgot to remove it.
Immediately removed